target/ppc/fpu_helper.c

```diff
@@ -383,6 +383,21 @@ static inline void float_inexact_excp(CPUPPCState *env)
     }
 }
 
+static inline void float_inexact_excp2(CPUPPCState *env)
```

Comment:
This function is just a placeholder to avoid changing the original one and break other functions that were not addressed yet. The idea is to remove the old implementation when everything is converted to use `update_fpscr`